parm/ccpp_v16beta_flake_c96.nml.IN

```diff
@@ -0,0 +1,338 @@
+﻿&amip_interp_nml
```

Comment:
Could this parm file be joined with the existing GFS v16 beta parm file, using variable substitutions?